docs(SECURITY): sync security policy (#607)
mdn/webextensions-examples: SECURITY.md

# Security Policy

## Reporting a Vulnerability

If you've discovered a security issue, please report it through the form linked
below, which will create a secure, private ticket.

> https://bugzilla.mozilla.org/form.web.bounty

Reviewer: ```suggestion
If you've discovered a security issue, report it using the linked form,
which creates a secure, private ticket.
https://bugzilla.mozilla.org/form.web.bounty
```

MDN may be eligible for
[Mozilla's Security Bug Bounty Program](https://www.mozilla.org/en-US/security/bug-bounty/).
You can find more information about the bounty program in the
[Mozilla Web Bug Bounty FAQ](https://www.mozilla.org/en-US/security/bug-bounty/faq-webapp/).

> You can use the above form even if you are not interested in a bounty reward.

Reviewer: ```suggestion
You can use the form even if you're not interested in a bounty reward.
```

# Security Policy

## Overview

This policy applies to MDN's website (`developer.mozilla.org`), backend services, and GitHub repositories in the [`mdn`](https://github.com/mdn) organization. Issues affecting other Mozilla products or services should be reported through the [Mozilla Security Bug Bounty Program](https://www.mozilla.org/en-US/security/bug-bounty/).

> For non-security issues, please file a [content bug](https://github.com/mdn/content/issues/new/choose), a [website bug](https://github.com/mdn/fred/issues/new/choose) or a [content/feature suggestion](https://github.com/mdn/mdn/issues/new/choose).

Reviewer: ```suggestion
For non-security issues, please file a [content bug](https://github.com/mdn/content/issues/new/choose), a [website bug](https://github.com/mdn/fred/issues/new/choose), or a [content or feature suggestion](https://github.com/mdn/mdn/issues/new/choose).
```
